src/Controller/IncidentController.php

```diff
@@ -15,19 +15,24 @@ class IncidentController extends AbstractController
     public function index(Request $request, RequestStack $requestStack, HttpClientInterface $apiClient): Response
     {
         // if not logged in, redirect to login page
-        // if ($requestStack->getSession()->get('is_logged_in') !== true) {
-        //     return $this->redirectToRoute('app_login_index');
-        // }
-
+        if ($requestStack->getSession()->get('is_logged_in') !== true) {
+            return $this->redirectToRoute('app_login_index');
+        }
         // call api client
-        $incidents = $apiClient->request('GET', '/signals/v1/private/signals', [
+        $incidents = $apiClient->request('POST', 'https://diensten.rotterdam.nl/sbmob/api/msb/openmeldingen', [
```

Comment:
zet `https://diensten.rotterdam.nl/sbmob/api/` in de .env en maak hier de url tot `msb/openmeldingen`
idem voor andere urls in deze pagina :-)

zo kunnen we voor de acc enzo gewoon een andere base url gebruiken

Reply:
Ik zie deze nu. Is goed. Ik vindt sowieso dat we over moeten naar een GET met queryparams.